Makefile

```diff
@@ -120,33 +119,27 @@ sample-compose-up: sample-compose-down
 sample-compose-down:
 	docker-compose -f test_scenarios/${SPLUNK_COMPOSE} down --volumes --remove-orphans || true
 
-test: clean ansible test_setup test_runner test_collection_cleanup
+test: clean ansible test_setup run_tests_centos7 run_tests_debian9
 
-test_helper_container:
-	@echo 'Starting container to run tests...'
-	docker run -d --rm --name=${TEST_IMAGE_NAME} --net=host -v /var/run/docker.sock:/var/run/docker.sock -v $(shell pwd):$(shell pwd) --entrypoint /bin/sh python:2.7.15-alpine3.7 -c 'tail -f /dev/null'
+test_centos7: clean ansible test_setup run_tests_centos7
 
-	@echo 'Install test requirements'
-	docker exec -i ${TEST_IMAGE_NAME} /bin/sh -c "pip install -r $(shell pwd)/tests/requirements.txt --upgrade"
-
-	@echo 'Running the super awesome tests'
-	mkdir test-results/pytest
-	docker exec -i ${TEST_IMAGE_NAME} /bin/sh -c "cd $(shell pwd); pytest -sv tests/ --junitxml test-results/pytest/results.xml"
+test_debian9: clean ansible test_setup run_tests_debian9
 
-test_collection_container_cleanup:
-	docker cp ${TEST_IMAGE_NAME}:$(shell pwd)/testresults.xml testresults.xml || echo "no testresults.xml"
+run_tests_centos7:
+	@echo 'Running the super awesome tests; CentOS 7'
+	pytest -sv tests/test_centos_7.py --junitxml test-results/centos7-result/testresults_centos7.xml
 
 test_setup:
 	@echo 'Install test requirements'
 	pip install -r $(shell pwd)/tests/requirements.txt --upgrade
+	mkdir test-results
 	mkdir test-results/pytest
```

Comment:
if you're making subdir's for centos / debian, do you need this pytest dir?

Reply:
nope lol